t/fcm-add-trac-env/00-basic.t

```diff
@@ -62,7 +62,7 @@ for NAME in bus car lorry taxi; do
     # Subversion repository paths in place
     if [[ -d "srv/svn/$NAME" ]]; then
         file_grep "$TEST_KEY-repository_dir" \
-            "repository_dir=$PWD/srv/svn/$NAME" \
+            "dir=$PWD/srv/svn/$NAME" \
```

Comment:
Not related to our issue here, but this test is failing on Travis CI, probably due to a newer version of Trac being made available recently. (The setting is now `.dir` instead of `repository_dir`. Shortening the match string should allow it to pass in Travis CI as well as our other environments.